compiler/rustc_data_structures/src/aligned.rs

```diff
@@ -15,10 +13,10 @@ pub const fn align_of<T: ?Sized + Aligned>() -> Alignment {
 /// # Safety
 ///
 /// `Self::ALIGN` must be equal to the alignment of `Self`. For sized types it
-/// is [`mem::align_of<Self>()`], for unsized types it depends on the type, for
```

Comment:
put turbofish here

Reply:
I did add a turbofish. It's now `align_of::<Self>()`. You replied on the deleted line, so maybe you missed that?